src/Configuration/RetryConfiguration.cs

```diff
@@ -29,7 +29,7 @@ public class RetryConfiguration
         /// Default is 250, 500, 1000.
         /// </summary>
         [ConfigurationKeyName("delays")]
-        public int[] DelaysMilliseconds { get; set; } = new[] { 750, 1200, 2500 };
```

Comment:
Any reason to remove the default values? Without the default, many Polly blocks would not perform any retries.

Reply:
with the default and entries in appSettings the values were doubling up, Ive made a constructor, that adds the values if not present, it might still double them up though !

Reply:
How about adding the default values only when it's empty?